src/routes/index.js

```diff
@@ -70,14 +70,27 @@ const routes = {
   '*': Home
 };
 
+const judgeNeedAuth = route => {
+  if (route.startsWith('/:dash')) {
+    return true;
+  }
+
+  const paths = ['/runtimes', '/purchased', '/profile', '/ssh_keys'];
+  if (paths.includes(route)) {
```

Comment:
simplify it: `return paths.includes(route)`

Reply:
ok.